daphne_worker/src/config.rs

```diff
@@ -940,6 +988,74 @@ impl<'srv> DaphneWorker<'srv> {
     pub(crate) fn greatest_valid_report_time(&self, now: u64) -> u64 {
         now.saturating_add(self.config().global.report_storage_max_future_time_skew)
     }
+
```

Comment:
Reviewer note: I verified that this was correctly refactored from `DapLeader::send_http_post()`.